src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -92,12 +92,7 @@ protected override void EndProcessing()
                         // The first input string does not represent a complete Json Syntax. 
                         // Hence consider the the entire input as a single Json content.
                     }
-#if CORECLR
-                    catch (Newtonsoft.Json.JsonSerializationException)
```

Comment:
I like this change, so when using `JsonObject.ConvertFromJson` in C#, you don't need to catch different exceptions between FullCLR and CoreCLR.